### PR TITLE
Fix the organization of Nextra

### PR DESCRIPTION
**Pull request by jmintuitive**
### Description

Moved the pages into the New Docs Format. 

Wrote a script to automatically fix links called `update-relative-links.py`. Used it in conjunction with `markdown-link-checker` to verify that once moved all the relative links worked. (Still some external links which need to be fixed, and a couple of pages which need to be migrated - see below). 

Remaining issues to be resolved in future PRs:

### Pages that still need migrating (Overall, not in this PR - but they were broken links)

- [ ]  [first-move-module.md](http://first-move-module.md/)
- [ ]  [gas-profiling.md](http://gas-profiling.md/)
- [ ]  [first-transaction.md](http://first-transaction.md/)
- [ ]  [first-coin.md](http://first-coin.md/)
- [ ]  [first-move-module.md](http://first-move-module.md/)
- [ ]  [remix-ide-plugin.md](http://remix-ide-plugin.md/)

### Remaining TODOs on migration effort

- [ ]  Adding a reference to “Standards” to the top-level (in addition to the Smart Contracts section.
- [ ]  Write an Advanced Guides index page.
- [ ]  local-network.md in SDK section
    - [ ]  TODO: As part of SDK → This is specifically referring to the one that is showing how the SDK can run a local network without the CLI.
- [ ] Remove the "TODOs" to fix links. (Didn't see a great search to find them all before I was heading to bed - @gregnazario do you know the various formats you were using for this?)

- Do all Lints pass?
  - [] Have you ran `pnpm spellcheck`?
  - [X] Have you ran `pnpm fmt`?
  - [X] Have you ran `pnpm lint`?

### CODE REVIEW

**Review by gregnazario — COMMENTED** [review]
lgtm, let's fix the html files that got converted to mdx

**Review by hariria — COMMENTED** [review]
Thanks for the hard work on this. Couple things I noticed that may need some closer attention:

1. I think the links on the landing page need to be updated as well. i.e., the links in `GET STARTED WITH THESE MOVE EXAMPLES` for each section (Coins, Objects, etc.) as well as the `DISCOVER BLOCKCHAIN FEATURES ON APTOS` section
2. We need to find another place to put the `Wallet` markdown doc in `Standards`

**Comment by jmintuitive** [comment]
> 2\. We need to find another place to put the `Wallet` markdown doc in `Standards`

Why do we need to find another place to put the `Wallet` markdown doc?

**Comment by hariria** [comment]
> > 2\. We need to find another place to put the `Wallet` markdown doc in `Standards`
> 
> 
> 
> Why do we need to find another place to put the `Wallet` markdown doc? 

The Wallet standards are unrelated to Smart Contracts

**Comment by jmintuitive** [comment]
nit: I noticed when merging that we're adding  → to every Card.Title. That seems really easy to forget over time, plus a bit annoying to find that character to copy + paste when adding new cards - @hariria would it make sense to add that to the `Card.Title` component instead? (With an optional flag to turn off potentially).

**Review by hariria — COMMENTED** [review]
Yep I can modify Card.Title in a future PR to take in a -> arrow prop.

File: apps/nextra/pages/en/build/sdks.mdx [hunk 1]
```diff
@@ -1,11 +1,7 @@
----
-title: "SDKs"
----
-
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks.mdx:4]: Could we leave this here?

File: apps/nextra/pages/en/build/_meta.tsx [hunk 2]
```diff
@@ -1,10 +1,9 @@
 export default {
   "quick-start": "Quick Start",
-  move: "Move",
+  "smart-contracts": "Smart Contracts",
```
[the remark this answers is not in the document]
Comment (jmintuitive) [reply apps/nextra/pages/en/build/_meta.tsx:3]: Can you follow up with me async about this change? I think that smart-contracts is a more useful name for discovery for folks who don't know what Move is, and folks who do know what Move is will equally know this is the right section for them.

**Review by hariria — APPROVED** [review]

File: apps/nextra/pages/en/build/_meta.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (gregnazario) [reply apps/nextra/pages/en/build/_meta.tsx:3]: Let's just get this change in, and we can talk after about it.

**Pull request merged**